includes/agentic-commerce/class-wc-stripe-agentic-commerce-product-meta-box.php

```
@@ -0,0 +1,202 @@
+	 *
+	 * @param int $product_id Product post ID.
+	 */
+	public function save_meta( int $product_id ): void {
+		if ( $product_id <= 0 ) {
+			return;
+		}
+
+		// The toggle is hidden while the merchant has the feature off, so the
+		// unchecked checkbox isn't posted — bail before that absence clobbers a
+		// stored 'yes' that should hold until they turn it back on.
+		if ( ! WC_Stripe_Agentic_Commerce_Integration::is_merchant_enabled() ) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Agentic_Commerce_Integration' in method 'save_meta'. (undefined)

(StaticAccess)